src/celeritas/global/ActionLauncher.device.hh

```diff
@@ -63,7 +63,8 @@ namespace celeritas
 template<class F>
 class ActionLauncher
 {
-    static_assert((std::is_trivially_copyable_v<F> || CELERITAS_USE_HIP)
+    static_assert((std::is_trivially_copyable_v<F> || CELERITAS_USE_HIP
+                   || __clang__)
```

Comment:
Guess this is just an undefined identifier rather than 0 since it's not part of a preprocessor expression